gateway-ha/etc/config.properties

```diff
@@ -0,0 +1,3 @@
+node.id=ffffffff-ffff-ffff-ffff-ffffffffffff
+node.environment=test
```

Comment:
The gateway is not clustered, why does it need a `node.id` and `node.environment`?

Reply:
That's because HttpServerModule depends on NodeInfo which needs NodeConfig which requires non-null node.environment and node.id.
I should be able to set these in Java and eliminate these from the config file.
Might need to dig deeper to see if there are drawbacks of doing it.

Reply:
There is no harm in having these value though .. right?